src/Toggle/Entity/MarketingToggle.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Volodymyr
+ * Date: 07/08/2015
+ * Time: 09:40
+ */
+
+namespace Clearbooks\Labs\Toggle\Entity;
+
+
+interface MarketingToggle
```

Comment:
```MarketingToggle``` is confusing.. it implies that it's a toggle for Marketing..

On the other hand

```MarketableToggle``` Implies what it is, a Toggle that is marketable.